pkg/util/arangod/client.go

```diff
@@ -200,6 +218,22 @@ func createArangodHTTPConfigForDNSNames(ctx context.Context, cli corev1.CoreV1In
 	return connConfig, nil
 }
 
+// createArangodVSTConfigForDNSNames creates a go-driver VST connection config for a given DNS names.
+func createArangodVSTConfigForDNSNames(ctx context.Context, cli corev1.CoreV1Interface, apiObject *api.ArangoDeployment, dnsNames []string) (http.ConnectionConfig, error) {
+	scheme := "vst"
+	if apiObject != nil && apiObject.Spec.IsSecure() {
+		// TODO set security for transport
+	}
+	transport := vst.NewTransport(/* TODO configure transport */)
```

Comment:
@ewoutp I haven't found examples of how to configure the transport correctly for secure vs. non-secure. Do you have a link to somewhere this is done?

Reply:
https://github.com/arangodb/go-driver/blob/master/vst/connection.go#L96

that line is what makes this work